package.json

```diff
@@ -10,12 +10,16 @@
     "pretest": "npm run lint && npm run build",
     "test": "floss --path test",
     "unit-test": "floss --path test",
-    "docs": "jsdoc -c jsdoc.conf.json -R README.md",
+    "docs": "mkdirp dist && jsdoc -c jsdoc.conf.json -R README.md",
+    "types": "npm run types:legacy && npm run types:default",
+    "types:default": "mkdirp dist/types/pixi.js && jsdoc -c types/jsdoc.conf.json && node types/assemble pixi.js",
+    "types:legacy": "mkdirp dist/types/pixi.js-legacy && jsdoc -c types/jsdoc-legacy.conf.json && node types/assemble pixi.js-legacy",
```

Comment:
It makes sense, however we have no userbase that use mini-modules yet, and there's no tutorial on how to make your own analogue of `bundle.js` that is required for that case.

Reply:
We have no userbase that uses individual packages yet, because they are not released.

Without doing it the way I'm describing it isn't even optional, it is just unsupported. If we are going to publish separate packages to npm (we are) then they should work.

Reply:
Hey guys!

I just ran into some typing problems with the new v5 release. We use TypeScript internally for everything so naturally having working types for Pixi is a must. One of the biggest cool new features for v5 is the individual packages which we can leverage to reduce our bundle size.

Unfortunately as discussed here there are no typings for them. It seems pixi-sound has ran into this issue as well since there is a "hackyish" workaround for it: https://github.com/pixijs/pixi-sound/tree/master/typings

On top of that I ran into some problems with the types in pixi-sound that I reported: https://github.com/pixijs/pixi-sound/issues/107

So I guess my main question is that since pixi-sound and pixi-spine are already written in TypeScript, what is your general feeling towards it? I am playing around with the idea of doing a fork of Pixi to convert it to TypeScript and use the TypeScript compiler to auto-generate the declarations (with comments included so you get nice tooltips inside IDEs like Visual Studio Code). This is essentially how all of our internal packages work as well.